java/src/apps/ActionListBundle.properties

```diff
@@ -44,7 +44,7 @@ jmri.jmrit.beantable.OBlockTableAction = Open OBlock Table
 jmri.jmrit.beantable.SectionTableAction = Open Section Table
 jmri.jmrit.beantable.TransitTableAction = Open Transit Table
 jmri.jmrit.beantable.AudioTableAction = Open Audio Table
-jmri.jmrit.beantable.IdTagTableAction = Open ID Tag Table
+jmri.jmrit.beantable.IdTagTableAction = Open Id Tag Table
```

Comment:
"ID" is more common than "Id" (at least in American English), so this should probably remain uppercase.